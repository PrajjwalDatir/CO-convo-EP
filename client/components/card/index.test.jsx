import React from 'react';
import { mount } from 'enzyme';
import Card from './index';

describe('<Card />', () => {
  it('should match snapshot', () => {
    const wrapper = mount(<Card />);
    expect(wrapper).toMatchSnapshot();
  });
});
