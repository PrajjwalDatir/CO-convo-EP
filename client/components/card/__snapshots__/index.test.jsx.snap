// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> should match snapshot 1`] = `
<Card
  className=""
>
  <div
    className="rounded bg-white p-10 shadow-lg "
  />
</Card>
`;
